projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -63,15 +70,11 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
   }
 
   ngOnInit(): void {
-    this.subscription = this.calendar.calendarOpen.subscribe((value: boolean) => {
-      if (value) {
         this.selectedDate = this.calendar.selectedDate;
         this.initializeDate();
-      }
-      this.opened = value;
-    });
-
-    this.dateAdapter.setLocale(this.locale);
+        this.appendToCalender();
```

Comment:
We should move this method call into an `AfterViewChecked` call like so:
```ts
ngAfterViewChecked(): void {
  this.appendToCalender();
}
```

You'll need to add `AfterViewChecked` to the class implementation list. [See Angular's docs for an example](https://angular.io/api/core/AfterViewChecked#usage-notes)